tests/Application.IntegrationTests/FailingTests.cs

```diff
@@ -30,15 +30,16 @@ private class TestData : TheoryData<string, string, Options, StateError, string[
             public TestData()
             {
                 Add(SignCases());
+                Add(BufferCases());
             }
 
             private static IEnumerable<(string, string, Options, StateError, string[])> SignCases()
             {
                 return
                     from optimization in new[] {"--Oz"}
-                    from useSymbolicGarbage in new[] {false, true}
-                    from useSymbolicAddresses in new[] {false, true}
-                    from useSymbolicContinuations in new[] {false, true}
+                    from useSymbolicGarbage in new[] {true}
```

Comment:
From a bit of trial and error it appears that only have enough of a memory budget right now to run these for a single combination of the execution options. I think reducing the options permutations but having the buffer test case is better, as a temporary measure, as it will let us golf the Alpine image with a bit more confidence.